apps/web/src/validators/profile/languageInformation.ts

```diff
@@ -39,9 +39,10 @@ export function hasEmptyRequiredFields({
   verbalLevel,
 }: PartialUser): boolean {
   return !!(
-    (!lookingForEnglish && !lookingForFrench && !lookingForBilingual) ||
-    (lookingForBilingual &&
-      (isEmpty(firstOfficialLanguage) || isEmpty(estimatedLanguageAbility))) ||
+    ((!lookingForEnglish && !lookingForFrench && !lookingForBilingual) ||
+      (lookingForBilingual &&
+        (isEmpty(firstOfficialLanguage) ||
+          isEmpty(estimatedLanguageAbility)))) ??
```

Comment:
I think this should be using ||.

Reply:
Updated in [ignore boolean primitives](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11651/commits/ccb9ba3ec73672fea328bb97fecdb8b1c1db5b91)